pkg/kubectl/cmd/taint.go

```diff
@@ -366,23 +382,20 @@ func validateNoTaintOverwrites(obj runtime.Object, taints []v1.Taint) error {
 }
 
 // updateTaints updates taints of obj
```

Comment:
@ravisantoshgudimetla `updateTaints applies taints to a node in  a clusters and returns a human understandable description of the resulting operation.`